clar2wasm/src/wasm_generator.rs

```diff
@@ -1633,8 +1635,9 @@ mod misc_tests {
 
     #[test]
     fn end_of_standard_data_is_correct() {
-        let standard_lib_wasm =
-            std::fs::read("src/standard/standard.wasm").expect("Failed to read WASM file");
+        let standard_lib_path =
+            format!("{}/src/standard/standard.wasm", env!("CARGO_MANIFEST_DIR"));
+        let standard_lib_wasm = std::fs::read(standard_lib_path).expect("Failed to read WASM file");
```

Comment:
No need to generate a `String` at runtime if all parts are know at compile-time

```suggestion
        const STANDARD_LIB_PATH: &str =
            concat!(env!("CARGO_MANIFEST_DIR"), "/src/standard/standard.wasm");
        let standard_lib_wasm = std::fs::read(STANDARD_LIB_PATH).expect("Failed to read WASM file");
```